python/pylibraft/pylibraft/test/test_random.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import numpy as np
+
+import rmm
+
+from pylibraft.random import rmat
+
+
+class TestDeviceBuffer:
```

Comment:
Since we're using this in two places now, we should probably centralize it in its own file so we can update it in one place when needed.

Reply:
Done